docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -164,7 +164,9 @@ Timers
 Name                                                Description
 =================================================== ========================================================================
 ``dagrun.dependency-check.<dag_id>``                Milliseconds taken to check DAG dependencies
-``dag.<dag_id>.<task_id>.duration``                 Seconds taken to finish a task
+``dag.<dag_id>.<task_id>.duration``                 Seconds taken to run a task
+``dag.<dag_id>.<task_id>.scheduled_duration``       Seconds a task spends in the Scheduled state, before being Queued
+``dag.<dag_id>.<task_id>.queued_duration``          Seconds a task spends in the Queued state, before being Running
```

Comment:
(Non-blocking) Just curious, are these lines manual changes or from some automated change?

Reply:
I added them manually, there is nothing enforcing sync between this doc and the code.

Reply:
Alright, that was what I thought.  I went looking for something like this when I started on the OTel work and didn't find it.  That's buried pretty deep.  :+1: